deployments/deployment.yaml

```diff
@@ -332,7 +332,7 @@ objects:
             inheritEnv: true
             command:
               - /jobs
-              - rename_domains
+              - retry-failed-tasks
```

Comment:
oh sorry i think the arg here should be `retry_failed_task` (singular with underscores)

```
bhouse@mac:content-sources-backend $ go run cmd/jobs/main.go retry_failed_task
9:09AM WRN Updated 0 tasks
```
```
bhouse@mac:content-sources-backend $ go run cmd/jobs/main.go retry-failed-tasks
9:09AM WRN Usage: go run cmd/jobs/main.go  $JOB_NAME
  (Possible jobs: retry_failed_task)
exit status 255
```

Reply:
![doh](https://i.giphy.com/media/v1.Y2lkPTc5MGI3NjExNm00M3l1aXo5bm1vZW9tem43OXZoYjAyb3Y5emUzOWE4YzE2a3BudSZlcD12MV9pbnRlcm5hbF9naWZfYnlfaWQmY3Q9Zw/xT5LMpBWU96Yivdfeo/giphy.gif)

Reply:
fixed